testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSDualWalletTest.scala

```diff
@@ -0,0 +1,65 @@
+package org.bitcoins.testkit.wallet
+
+import org.bitcoins.db.AppConfig
+import org.bitcoins.server.BitcoinSAppConfig
+import org.bitcoins.testkit.BitcoinSTestAppConfig
+import org.bitcoins.testkit.wallet.DLCWalletUtil.InitializedDLCWallet
+import org.bitcoins.testkit.wallet.FundWalletUtil.FundedWallet
+import org.bitcoins.wallet.config.WalletAppConfig
+import org.scalatest.FutureOutcome
+
+trait BitcoinSDualWalletTest extends BitcoinSWalletTest {
+  import BitcoinSWalletTest._
```

Comment:
is the `import` necessary given the extension?

Reply:
Yes, I am not sure why though

Reply:
Oh I get it, it is in the companion object which is what is being imported here